rules/rollup.go

```diff
@@ -203,3 +213,63 @@ func bytesArrayCopy(values [][]byte) [][]byte {
 	}
 	return result
 }
+
+func (t rollupTarget) Schema() (*schema.RollupTarget, error) {
```

Comment:
Can we move the methods closer to where the other methods are defined for this type? Makes it easier to read/find stuff.